README.md

```diff
@@ -37,24 +37,24 @@ XUnion uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/).
 ## Command-Line Interface
 
 ```bash
-~/xunion/bin $ ./xucli placeorder --price 0.12 --quantity 5
+~/xud/bin $ ./xudcli placeorder --price 0.12 --quantity 5
 {"fieldCount":0,"affectedRows":1,"insertId":1,"serverStatus":2,"warningCount":0,"message":"","protocol41":true,"changedRows":0}
-~/xunion/bin $ ./xucli getorders
+~/xud/bin $ ./xudcli getorders
 {"bids":[{"price":0.12,"quantity":5}],"asks":[]}
 ```
 
 ## Configuration
 
-The configuration file uses [TOML](https://github.com/toml-lang/toml) and by default is located at  `~/.xunion/xunion.conf` on Linux or `AppData\Local\XUnion\xunion.conf` on Windows. Default settings which can be overridden are shown below.
+The configuration file uses [TOML](https://github.com/toml-lang/toml) and by default is located at  `~/.xud/xud.conf` on Linux or `AppData\Local\xud\xud.conf` on Windows. Default settings which can be overridden are shown below.
```

Comment:
To follow the convention for windows AppData folders, lets stick with `AppData\Local\Xud`.